src/main/java/com/ciaranmckenna/readinglistapp/util/Mapper.java

```diff
@@ -0,0 +1,25 @@
+package com.ciaranmckenna.readinglistapp.util;
+
+import com.ciaranmckenna.readinglistapp.dao.entity.Author;
+import com.ciaranmckenna.readinglistapp.dao.entity.Book;
+import com.ciaranmckenna.readinglistapp.dto.AuthorRecord;
+import com.ciaranmckenna.readinglistapp.dto.BookRecord;
+
+public class Mapper {
```

Comment:
Is this class to reduce the times I have mapped entities to records throughout the code base?

Reply:
that is right, you create a util, that maps the Entities to record, and you dont have to repeat yourself with creating a new record always, and using method reference, the map is more easy ... and single responsibility